tests/set/set4.c

```diff
@@ -35,9 +35,23 @@ int *next_int(void) {
 int main(void) {
 	struct set *s = set_create(NULL, cmp_int);
 	size_t i;
+	int **plist;
+
+	plist = malloc(5000 * sizeof *plist);
```

Comment:
Prefer `calloc` for array allocations.

Reply:
Hiya, we don't tend to calloc unless there's an explicit need to set items to all-bytes-zero. Pointer and size arithmetic is absolutely everywhere in C, I don't think it's fair to try to convey all array-like uses with calloc.